compiler/luci/pass/src/FusePreActivationBatchNormPass.cpp

```diff
@@ -114,11 +114,157 @@ bool is_batchnorm_add(const luci::CircleAdd *add, luci::CircleMul *&mul, luci::C
   return true;
 }
 
+luci::CircleConv2D *get_forward_conv2d(luci::CircleNode *node, uint32_t channel_size)
+{
+  auto opcode = node->opcode();
+  if (opcode == luci::CircleOpcode::CONV_2D)
+  {
+    auto conv = loco::must_cast<luci::CircleConv2D *>(node);
+    auto filter = dynamic_cast<luci::CircleConst *>(conv->filter());
+
```

Comment:
maybe remove this empty line to follow like below `else` block or add space in `else` block

Reply:
Updated. Thanks.